style/base.css

```diff
@@ -202,30 +184,32 @@ input::placeholder {
   height: 100%;
 }
 
-.sidebar-widget .sidebar-list.sidebar-right {
+.jlab-gather-sidebar-widget
+  .jlab-gather-sidebar-list
+  .jlab-gather-sidebar-right {
```

Comment:
`sidebar-list` and `sidebar-right` is also a double class selector so there needs to not be a space.